scripts/signin.js

```diff
@@ -1,84 +1,136 @@
-import { ADMIN_EMAIL, ADMIN_PASSWORD } from "./base.js";
+import {
+  INVALID_CLASS,
+  ADMIN_EMAIL,
+  ADMIN_PASSWORD,
+  FOLDER_URL,
+  API_SIGN_IN,
+  HTTP_STATUS_OK,
+  HTTP_STATUS_BAD_REQUEST,
+} from "./constants.js";
 import { isValidEmail } from "./inputValidation.js";
 import { togglePasswordVisibility } from "./togglePasswordVisibility.js";
 
-document.addEventListener("DOMContentLoaded", function () {
-  const emailInput = document.querySelector("#email");
-  const passwordInput = document.querySelector("#password");
-  const loginButton = document.querySelector(".base-button");
-  const togglePasswordIcon = document.querySelector(
-    "#togglePasswordVisibility"
-  );
+const emailInput = document.querySelector("#email");
+const passwordInput = document.querySelector("#password");
+const loginButton = document.querySelector(".base-button");
+const togglePasswordIcon = document.querySelector("#togglePasswordVisibility");
+const accessToken = localStorage.getItem("accessToken");
 
-  // 이메일 입력 처리 함수 : 유효성 검사 및 에러 메시지 관리
-  function handleEmailInput() {
-    const emailValue = emailInput.value;
-    const emailError = document.querySelector("#emailError");
+// 로그인 페이지에 접근시 로컬 스토리지에 accessToken이 있는 경우 “/folder” 페이지로 이동
+if (accessToken) {
+  window.location.href = FOLDER_URL;
+}
 
-    // 이메일 입력부분이 비어 있는 경우
-    if (!emailValue) {
-      emailError.textContent = "이메일을 입력해 주세요.";
-      emailInput.classList.add("invalid");
-      // 이메일 입력부분이 비어있지 않는 경우 + 이메일 형식이 유효한지 확인 후 유효하지 않은 경우
-    } else if (!isValidEmail(emailValue)) {
-      emailError.textContent = "올바른 이메일 주소가 아닙니다.";
-      emailInput.classList.add("invalid");
-      // 이메일 입력부분이 비어있지 않은 경우 + 이메일 형식이 유효한 형식인 경우
-    } else {
-      emailError.textContent = "";
-      emailInput.classList.remove("invalid");
-    }
+// 이메일 입력 처리 함수 : 유효성 검사 및 에러 메시지 관리
+function handleEmailInput() {
+  const emailValue = emailInput.value;
+  const emailError = document.querySelector("#emailError");
+
+  // 이메일 입력부분이 비어 있는 경우
+  if (!emailValue.trim()) {
```

Comment:
trim을 변수에 할당할때 해도 좋을 것 같아요.

```suggestion
  const emailValue = emailInput.value.trim();
  const emailError = document.querySelector("#emailError");

  // 이메일 입력부분이 비어 있는 경우
  if (!emailValue) {
```